src/libraries/System.Net.Http/src/System/Net/Http/Headers/QPackStaticTable.cs

```diff
@@ -96,7 +96,7 @@ internal static class QPackStaticTable
             (new HeaderDescriptor(KnownHeaders.Authorization), ""), // 84
             (new HeaderDescriptor(KnownHeaders.ContentSecurityPolicy), "script-src 'none'; object-src 'none'; base-uri 'none'"), // 85
             (new HeaderDescriptor("early-data"), "1"), // 86
-            (new HeaderDescriptor("expect-ct"), ""), // 87
+            (new HeaderDescriptor(KnownHeaders.ExpectCT), ""), // 87
```

Comment:
Should we be adding KnownHeaders entries for the other `new HeaderDescriptor(string)` entries in this table?

Reply:
That would be my hope as part of #63750, I updated the post to include `QPackStaticTable`